### ISSUE TITLE
nwg-menu hidden by full-screen windows

### ISSUE THREAD

**Issue by cjns1989** [nwg-piotr/nwg-menu#10]
I use the WayfireWM compositor and I noticed that when I open nwg-menu on a desktop/workspace where a window is displayed full screen the menu is not visible. It is open and I can see it via an **<kbd>Alt</kbd> + <kbd>Tab</kbd>** keyboard action (or by switching to another workspace) but I am unable to access it on the workspace where I opened the menu. 

It looks like the corresponding window is at the bottom of the stack when it should be at the top and focused so the user is able to interact (type in the search field... or tab through the entries... or click on an entry... etc.)

Note, that I have no idea who is misbehaving in this instance — is this an nwg-menu problem or a WayfireWM problem?

**Comment by nwg-piotr** [comment]
Haven't used Wayfire for several months, so I'm not sure. On sway fullscreen is fullscreen: covers everything.

**Comment by cjns1989** [comment]
I get your point. Due to some problems with the Wayfire panel I started using full screen instead of maximize. The difference being apparently that with full screen the decorations (titlebar and such... as well as menu bars etc.) are removed and the useful part of the window occupies the entire screen while in maximized mode the original layout of the window is left as is and the window is only stretched so as to occupy all the available display area... So if you have a panel that is not set to autohide the WM will leave well alone and display your stretched window above or below it using whatever space is left (i.e. the entire screen minus what's used up by the panel). On a Windows desktop and look-alikes you would hit F11 to switch to full screen mode and click on something resembling a small square (next to the _ that is supposed to mean minimize) or use the 'Maximize' option in the  menu in order to maximize the window. 

Now I eventually discovered that with Wayfire you need to use the 'grid' plugin to set a keyboard shortcut that lets you maximize the focused window as opposed to 'full-screening' it.

In other words I was confused and everything appears to be working as planned.

**Comment by nwg-piotr** [comment]
BTW, I need a fresh install of Wayfire. What's on my disk is heavily out of date. I'm quite curious what's new.

**Comment by cjns1989** [comment]
I'm not involved in Wayfire in any way… just another Joe User looking for a barebones Wayland DE that I can adapt to my requirements… I would suggest you get in touch with the Wayfire developers e.g. via irc (#wayfire@libera.chat) or Matrix… I have found them to be very reactive and remarkably patient with outsiders.. as it happens two of the main weaknesses of the current version are the panel and the applications menu… ;-)

**Comment by nwg-piotr** [comment]
I know ammen99 and soreau. They do a great work, and their wf-shell misses nothing but more time. AFAIK, for now they are more focused on the compositor itself. But I haven't seen Wayfire for several months, so I may be wrong.
